src/components/Typography.tsx

```diff
@@ -2,23 +2,30 @@ import { ReactNode } from 'react'
 import { StyleSheet, Text, TextProps } from 'react-native'
 
 import Colors from '../themes/colors'
-import Fonts from '../themes/fonts'
 
 const styles = StyleSheet.create( {
   base: {
     color: Colors.PrimaryText,
   },
-  body: {},
+  body: {
+    fontSize: 17,
+    lineHeight: 29.75,
+    fontWeight: '400',
+  },
   button: {
     color: Colors.PrimaryText,
   },
-  header: {
-    fontSize: 18,
-    fontFamily: Fonts.NotoSansLight,
```

Comment:
Are we generally not using Noto anymore?

Reply:
Tracked in https://github.com/shabados/mobile/issues/194